tekton/predicates.go

```diff
@@ -38,7 +38,7 @@ func IntegrationPipelineRunPredicate() predicate.Predicate {
 		},
 		UpdateFunc: func(e event.UpdateEvent) bool {
 			return IsIntegrationPipelineRun(e.ObjectNew) &&
-				(hasPipelineRunStateChangedToStarted(e.ObjectOld, e.ObjectNew) || hasPipelineRunStateChangedToFinished(e.ObjectOld, e.ObjectNew))
+				(hasPipelineRunStateChangedToStarted(e.ObjectOld, e.ObjectNew) || hasPipelineRunStateChangedToFinished(e.ObjectOld, e.ObjectNew) || hasPipelineRunStateChangedToDeleting(e.ObjectOld, e.ObjectNew))
```

Comment:
```suggestion
				(hasPipelineRunStateChangedToStarted(e.ObjectOld, e.ObjectNew) || 
                                hasPipelineRunStateChangedToFinished(e.ObjectOld, e.ObjectNew) ||
				hasPipelineRunStateChangedToDeleting(e.ObjectOld, e.ObjectNew))
```

Reply:
Moved them into new lines and then `go fmt ./...` added its own magic.

Reply:
okay :(